apps/site/lib/site_web/views/teaser_view.ex

```diff
@@ -29,4 +35,31 @@ defmodule SiteWeb.CMS.TeaserView do
     |> Enum.reject(&is_nil/1)
     |> Enum.join(", ")
   end
+
+  @doc """
+  Teasers requested from the CMS context always use the ContentList
```

Comment:
as above re: "CMS context" (it's not super clear that this is referring to the boundary between code in the `cms` app within this project, and code in other apps)